### DISCUSSION TITLE
Developer Contact Information Request / Solicitud de Información de Contacto del Desarrollador 💻 (25 FEB 2024 @ 11:59PM)

### DISCUSSION THREAD

**Discussion by rukitoyordan** [uprm-inso4101-2023-2024-s2/semester-project-rumaradoptionapp#87] [Announcements]
EN 🇺🇸

Please provide the following contact information for better communication between parties as well as the possibility for calendarization of events/meetings.

> - Name, Last Name
> - Position and Team (Team Leader, Developer; What team you belong to)
> - Institutional Email
> - Phone Number (Optional)

SPA 🇪🇸

Proporcione la siguiente información de contacto para una mejor comunicación entre las partes, así como la posibilidad de programar eventos/reuniones.

> - Nombre, Apellido
> - Posición y equipo (líder del equipo, desarrollador; a qué equipo pertenece)
> - Correo Institucional
> - Numero de teléfono (opcional)

🚨 Deadline / Fecha Limite: _**25 Feb 2024 @ 2359 (11:59PM)**_

**Comment by rukitoyordan** [comment]
Ejemplo:

> Fabián Pérez Muñoz
> Manager; Human Profiles, Sign/Up Login, Website Creation
> fabian.perez4@upr.edu
> (787) 466-7241

**Reply by Angel-Rosado** [reply]
Ángel Rosado
Team leader: Human Profile
angel.rosado13@upr.edu
787-638-7129

**Comment by Javi510348** [comment]
Javier Torres Rivera
Developer; Login & Signup
javier.torres16@upr.edu
(787)-244-7169

**Comment by smbv** [comment]
Sofia M. Barrera Vega
Team Leader - Website Creation
sofia.barrera@upr.edu
(787)-579-8639

**Comment by JohnSantiagoRuiz** [comment]
John P. Santiago
Team Leader; Pet Listings
john.santiago12@upr.edu
(787) 241-5257

**Comment by Alejandro-DeJesus** [comment]
Alejandro J. De Jesus Claudio
Team Leader, Pet Profiles
alejandro.dejesus1@upr.edu
(787)-983-0401

**Comment by Adriana-Isabel** [comment]
Adriana I. Carrasquillo 
Member - Website Creation
adriana.carrasquillo1@upr.edu
(787)-664-4936

**Comment by frances-sepulveda** [comment]
Frances Sepúlveda 
Team Lead; Sign Up/Login
Frances.sepulveda1@upr.edu
(939)-281-4110

**Comment by yamayrie** [comment]
Yamayrie Lebrón Borrero
Member; Sign-up/login
yamayrie.lebron@upr.edu
787-383-2502

**Comment by jhsanchez26** [comment]
Juan Sanchez Ponte
Developer, Sign-up/Login
juan.sanchez13@upr.edu
787-436-5237

**Comment by rayanemili** [comment]
Rayane Mili
Developer, Pet Profiles
rayane.mili@upr.edu
939-608-5397

**Comment by Saul-Caban-UPRM** [comment]
Saúl E. Cabán Torres 
Developer, Sign-up/Login 
saul.caban@upr.edu
939-319-6896

**Comment by RaquelQuinones** [comment]
Raquel M. Quinones
Developer, Pet Profiles
raquel.quinones7@upr.edu
(939)-240-2296

**Comment by 1uismar33r0** [comment]
Luis A. Marrero Morales
Developer, Sign-up/Login
luis.marrero33@upr.edu
(939)-299-2533

**Comment by GiovanniMaysonet** [comment]
Giovanni Maysonet Rodriguez
Developer, Web Design
giovanni.maysonet@upr.edu
939-418-4115

**Comment by AlexisTraversoPR** [comment]
Alexis Traverso
Developer, Pet Profiles
alexis.traverso@upr.edu
787-249-3117

**Comment by ErickCruz4** [comment]
Erick Cruz
Developer, Human Profiles
erick.cruz4@upr.edu
787-222-6907

**Comment by HectorSepul** [comment]
Héctor R. Sepúlveda 
Developer, Human Profiles
hector.sepulveda5@upr.edu
939-777-3666
